### ISSUE TITLE
iOS UIWebview replacement to WKWebview

### ISSUE THREAD

**Issue by gurdeep2020** [marcuswestin/WebViewJavascriptBridge#415]
Hi there, 
               I am reporting an issue with regards to UIWebview api used in this project. As Apple has deprecated the api UIWebview and replaced this with WkWebview.  So any developer who will upload the iOS app to app store from December 2020 with UIWebview implementation will not be accepted. 
So kindly please provide an updated WebViewJavascriptBridge version without the reference and usage of UIWebview. It  will be greatly appreciated.  

# WVJB Bug Report

Thanks for reporting an issue with WebViewJavascriptBridge.

### Do these 4 things and I will fix your problem!

1. Go to https://github.com/marcuswestin/WebViewJavascriptBridge and click Fork.
2. Clone your fork, `cd` into it and run `make test`. All tests should pass!
3. Edit `Tests/WebViewJavascriptBridgeTests/BridgeTests.m` and create a new, failing test which demostrates your issue.
4. Create a pull request for https://github.com/marcuswestin/WebViewJavascriptBridge

#### That's it!

I will take it from there and promise that I'll fix your problem ASAP.

#### If you don't do this then I can't help you!

And I probably won't :)

Cheers,
@marcuswestin

**Comment by luoshihui** [comment]
you can download the code, then delete the WebViewJavascriptBridge.h/.m file, then add others to your project.

**Comment by ajabade** [comment]
Hello Team,

I'm using this lib for WKWebView but still, this library has UIWebView references. As per Apple, they won't accept the app store update for UIWebView after Dec 2020. 

Could you please remove references for UIWebView and update the library version?

Thank you so much!

**Comment by tc1993** [comment]
> 您可以下载代码，然后删除WebViewJavascriptBridge.h / .m文件，然后将其他文件添加到您的项目中。

yes  delete it and  import WKWebViewJavascriptBridge replace it。 it‘s ok

**Comment by ajabade** [comment]
Thank you for the update.
